Week2/Homework/assets/question-five.js

```diff
@@ -0,0 +1,9 @@
+let z = 7.25;
+console.log(z);
```

Comment:
Ah, I think you accidentally added the homework from week 2, I will ignore it :)

Reply:
Hello Marciano,
This was the rest of the 2 week hw. Yash asked us to deliver along with the week 3 hw.